tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -241,6 +242,93 @@ def test_eq_with_srid(self):
         self.assertEqual(p0, "SRID=0;POINT (5 23)")
         self.assertNotEqual(p1, "SRID=0;POINT (5 23)")
 
+    @skipIf(geos_version_tuple() < (3, 12), "GEOS >= 3.12.0 is required")
+    def test_equals_identical(self):
+        """
+        Test equals_identical comparaison
+        """
+        # empty inputs of different types are not equals_identical
+        g1 = fromstr("POINT EMPTY")
+        g2 = fromstr("LINESTRING EMPTY")
+        self.assertFalse(g1.equals_identical(g2))
```

Comment:
Use `assertIs(…, True/False)` for testing boolean values, rather than `assertTrue()` and `assertFalse()`, so you can check the actual boolean value, not the truthiness of the expression  as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
good to know 👍